bullish.js

```diff
@@ -199,15 +215,17 @@ module.exports = (server, opts, next) => {
       return Promise.reject(new Error(`${name} job was never defined`));
     }
 
-    const validationEnabled = (opts.validate !== false && hoek.reach(q, '_bullishConfig.validate'));
+    const validationEnabled = opts.validate !== false && hoek.reach(q, '_bullishConfig.validate');
     const pre = validationEnabled ? validate(q, data) : Promise.resolve(data);
 
-    return pre.then((data) => {
+    return pre.then(data => {
       return q._bullishHandler({
         jobId: `bullish-injected-${Date.now()}`,
         data,
         pre: opts.pre,
         simulated: opts.simulated || false,
+        progress: () => {}
+        //@todo: try to add the actual Job prototype
```

Comment:
Do you want to resolve this TODO before this PR gets merged?

Reply:
I just checked the prototype. Merging the prototypes is unrealistic, since it has quite some dependencies. I think for  the scope of this PR, this should be enough. I will remove the comment